hudi-aws/src/main/java/org/apache/hudi/aws/sync/AwsGlueCatalogSyncTool.java

```diff
@@ -65,6 +63,56 @@ public static void main(String[] args) {
     FileSystem fs = FSUtils.getFs(cfg.basePath, new Configuration());
     HiveConf hiveConf = new HiveConf();
     hiveConf.addResource(fs.getConf());
-    new AwsGlueCatalogSyncTool(cfg, hiveConf, fs).syncHoodieTable();
+    AwsGlueCatalogSyncTool.newBuilder()
```

Comment:
the builder pattern is good. but do you need it be in this PR? i was only expecting pom.xml changes :)

Reply:
I can move it to a separate PR. I just noticed and did the change. But, better to keep it in a separate PR just in case if we have to revert this PR then we only revert dependency changes.